tests/rptest/services/redpanda_installer.py

```diff
@@ -290,24 +294,32 @@ def start(self):
 
         self._started = True
 
-    def _initialize_released_versions(self):
+    @property
+    def released_versions(self):
         if len(self._released_versions) > 0:
-            return
-
-        # Initialize and order the releases so we can iterate to previous
-        # releases when requested.
-        releases_resp = requests.get(
-            "https://api.github.com/repos/redpanda-data/redpanda/releases")
-        releases_resp.raise_for_status()
-        try:
-            self._released_versions = [
-                int_tuple(VERSION_RE.findall(f["tag_name"])[0])
-                for f in releases_resp.json()
-            ]
-        except:
-            self._redpanda.logger.error(releases_resp.text)
-            raise
-        self._released_versions.sort(reverse=True)
+            return self._released_versions
```

Comment:
Just checking my python concurrency knowledge: this is safe because we're swapping in the list fully at L322, and python is serialized by the GIL such that single lines are atomic, right?

Reply:
right